lib/matplotlib/tests/test_agg.py

```diff
@@ -219,3 +219,17 @@ def test_too_large_image():
     buff = io.BytesIO()
     with pytest.raises(ValueError):
         fig.savefig(buff)
+
+
+def test_chunksize():
+    x = range(9000000)
+    fig = plt.figure()
+    ax = fig.add_subplot(1, 1, 1)
+
+    # Test with small chunksize
+    tmp = rcParams['agg.path.chunksize']
```

Comment:
You don't need to worry about this; the cleanup on the test will reset all `rcParams`.